NOTICE

```diff
@@ -21,12 +21,12 @@ Ohai incorporates functionality from Open4 (http://www.codeforpeople.com/lib/rub
 Chef Notices
 ============
 
-Developed at Opscode (http://www.opscode.com).
+Developed at Chef Software Inc. (http://www.chef.io).
```

Comment:
I'd leave this as "Developed at CHEF". If that bothers you, "Developed at Chef". We only use "Chef Software, Inc." for legal purposes, we don't advertise as that.